src/electos/ballotmaker/ballots/ballot_layout.py

```diff
@@ -139,42 +138,45 @@ def build_ballot() -> str:
     )
     doc.addPageTemplates(three_column_template)
     doc.addPageTemplates(one_column_template)
-    # add a ballot contest to the second frame (colomn)
-    layout_1 = CandidateContestLayout(
-        CandidateContestData(spacetown_data.can_con_1)
-    )
-    layout_2 = CandidateContestLayout(
-        CandidateContestData(spacetown_data.can_con_2)
-    )
-    layout_3 = CandidateContestLayout(
-        CandidateContestData(spacetown_data.can_con_3)
-    )
-    layout_4 = CandidateContestLayout(
-        CandidateContestData(spacetown_data.can_con_4)
-    )
-    layout_5 = BallotMeasureLayout(
-        BallotMeasureData(spacetown_data.ballot_measure_1)
-    )
-    layout_6 = BallotMeasureLayout(
-        BallotMeasureData(spacetown_data.ballot_measure_2)
-    )
```

Comment:
👍🏼 to seeing this block turn into a loop :).

Reply:
It's like real software now!